### ISSUE TITLE
Incorrect indent introduced in 0.45.3

### ISSUE THREAD

**Issue by paulb777** [nicklockwood/SwiftFormat#697]
An extra two spaces of indent are added at lines 405-408 of https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/third_party/FirestoreEncoder/FirestoreDecoder.swift

```
 $ swiftformat --indent 2 --maxwidth 100 --wrapparameters afterfirst -disable sortedImports,unusedArguments,wrapMultilineStatementBraces --lint   Firestore/third_party/FirestoreEncoder/FirestoreDecoder.swift
Running SwiftFormat...
(lint mode - no files will be changed.)
/Users/paulbeusterien/gh9/firebase-ios-sdk/Firestore/third_party/FirestoreEncoder/FirestoreDecoder.swift:405:1: warning: (indent) Indent code in accordance with the scope level.
/Users/paulbeusterien/gh9/firebase-ios-sdk/Firestore/third_party/FirestoreEncoder/FirestoreDecoder.swift:406:1: warning: (indent) Indent code in accordance with the scope level.
/Users/paulbeusterien/gh9/firebase-ios-sdk/Firestore/third_party/FirestoreEncoder/FirestoreDecoder.swift:407:1: warning: (indent) Indent code in accordance with the scope level.
/Users/paulbeusterien/gh9/firebase-ios-sdk/Firestore/third_party/FirestoreEncoder/FirestoreDecoder.swift:408:1: warning: (indent) Indent code in accordance with the scope level.
```

**Comment by nicklockwood** [comment]
@paulb777 this appears to already be fixed in 0.45.3 (I can reproduce it in 0.45.2).

**Comment by paulb777** [comment]
Hmm, I'm seeing the opposite. Maybe we have a disconnect about what the right format is?

Shouldn't the right closing brace at line 408(411 in your [version](https://github.com/nicklockwood/SwiftFormat/blob/d294de84b97d0e4a63abaaa3ae92430313ce47f7/Snapshots/Issues/697.swift#L411)) be in the same column as the `p` of `public` on line 410(413)?

If not, why?

**Comment by nicklockwood** [comment]
@paulb777 ah, sorry, I misunderstood. I assumed that the file you had linked to was the correct version, not the one with the wrong indent already applied.

I suspect the braces here are being mistaken for a closure. I'll investigate and try to fix it in the next release.

**Comment by nicklockwood** [comment]
@paulb777 I pushed a fix to the develop branch. This will land in 0.45.5 (probably in a few days).

**Comment by nicklockwood** [comment]
@paulb777 fixed in 0.45.5.
